files/fr/web/html/global_attributes/enterkeyhint/index.html

```diff
@@ -0,0 +1,96 @@
+---
+title: enterkeyhint
+slug: Web/HTML/Global_attributes/enterkeyhint
+browser-compat: html.global_attributes.enterkeyhint
+translation_of: 'Web/HTML/Global_attributes/enterkeyhint'
+---
+<div>{{HTMLSidebar("Global_attributes")}}</div>
+
+<p><a href="/fr/docs/Web/HTML/Global_attributes">L'attribut universel</a> <strong><code>enterkeyhint</code></strong> 
+est un attribut à valeur contrainte définissant le libellé d'action (ou d'icône) à présenter pour la clé saisie sur le clavier virtuel.</p>
+
+<div>{{EmbedInteractiveExample("pages/tabbed/attribute-enterkeyhint.html","tabbed-shorter")}}</div>
+
+<h2 id="description">Description</h2>
+<p>Les <a href="/fr/docs/Learn/Forms">contrôles de formulaires</a> (tels que les éléments <a href="/fr/docs/Web/HTML/Element/textarea"><code>&lt;textarea&gt;</code></a>
+ou <a href="/fr/docs/Web/HTML/Element/input"><code>&lt;input&gt;</code></a>) ou les éléments utilisant l'attribut
+<a href="/fr/docs/Web/HTML/Global_attributes/contenteditable"><code>contenteditable</code></a> peuvent spécifier un attribut
+<a href="/fr/docs/Web/HTML/Global_attributes/inputmode"><code>inputmode</code></a> pour contrôler le type de clavier virtuel qui sera utilisé. Pour améliorer davantage l'expérience du visiteur, la clé à saisir peut être personnalisée spécifiquement en fournissant un attribut <code>enterkeyhint</code> indiquant comment la clé de saisie doit être libellée (ou quelle icône doit être affichée). La clé de saisie représente généralement ce que la personne va devoir faire par la suite. Les actions consistent généralement en l'envoi d'un texte, l'insertion d'une nouvelle ligne, le lancement d'une recherche, etc.</p>
```

Comment:
same as before about "la clé à saisir"

Reply:
fixed in https://github.com/mdn/translated-content/pull/1823/commits/047ee2186aa63fefe5382046869d2d4fcd5338ad